packages/esm-ugandaemr-app/src/pages/system-info/system-info.component.tsx

```diff
@@ -13,23 +13,21 @@ import {
   TableCell,
   Tile,
 } from '@carbon/react';
-import { ErrorState, UserHasAccess, showToast, showNotification } from '@openmrs/esm-framework';
-import { useGetSystemInformation, updatePropertyValue, useRetrieveFacilityCode } from './system-info.resources';
+import { ErrorState, UserHasAccess, showToast, showNotification, showSnackbar } from '@openmrs/esm-framework';
+import { updatePropertyValue, useGetSystemInformation, useRetrieveFacilityCode } from './system-info.resources';
 import styles from './system-info.scss';
 import coatOfArms from '../../images/coat_of_arms.png';
 import UpdateFacilityCode from './update-facility-code-button.component';
 import { PRIVILEGE_UPDATE_FACILITY_CODE } from '../../constants';
-import UpdateInternetPowerButton from '../internet-power/internet-power-button.component';
 
 interface FacilityCodeDetails {
   value?: string;
-  uuid?: string;
 }
 
 const OverallSystemInfo = ({ buildInfo, emrVersion, facilityCodeDetails, setFacilityCodeDetails }) => {
   const { t } = useTranslation();
   const buildDateTime =
-    Object.keys(buildInfo).length > 0
+    buildInfo && buildInfo['SystemInfo.OpenMRSInstallation.systemDate']
```

Comment:
move this to config or constants

Reply:
> move this to config or constants

This has been addressed in the subsequent commit